src/compiler/factory.ts

```diff
@@ -214,61 +214,107 @@ namespace ts {
             : node;
     }
 
-    // Type Elements
+    // Signature elements
 
-    export function createSignatureDeclaration(kind: SyntaxKind, typeParameters: TypeParameterDeclaration[] | undefined, parameters: ParameterDeclaration[], type: TypeNode | undefined) {

```

Comment:
I can't really tell from the diff if the code was merely re-ordered or if there are more substantive changes. Looks similar on the LHS and RHS though.